polyfill/lib/calendar.mjs

```diff
@@ -338,14 +385,6 @@ const jpn = {
   // Note: C locale era names available at
   // https://github.com/unicode-org/icu/blob/master/icu4c/source/data/locales/root.txt#L1582-L1818
 
-  compareDate(one, two) {
```

Comment:
This seems to be still used in line 393.

Reply:
Its removal was a complete accident in cutting and pasting other code! Thanks for catching. No tests caught it :'(